scripts/generate-google3-docs/resource-reference/overview.md

```diff
@@ -190,6 +190,10 @@ issues for {{product_name_short}}.
   <td><a href="/compute/docs/">{{compute_name}}</a></td>
   <td><a href="/config-connector/docs/reference/resource-docs/compute/computefirewallpolicyassociation.md">ComputeFirewallPolicyAssociation</a></td>
 </tr>
+<tr>
+  <td><a href="/compute/docs/">{{compute_name}}</a></td>
+  <td><a href="/config-connector/docs/reference/resource-docs/compute/computefirewallpolicyrule.md">ComputeFirewallPolicyRule</a></td>
```

Comment:
Not a problem, but why has this popped up here?  I'm confused why mockgcp would trigger this.  Did you add this manually?

Reply:
I added this manually because the resource document is missing from our resources overview: https://screenshot.googleplex.com/4LYei5wFDfKwBxg.png